vue-frontend/src/components/android-app-development/CtaSection.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="tw-container tw-relative tw-flex tw-flex-row tw-py-[2rem]">
+    <div class="tw-relative tw-z-[1]">
+      <img
+        :src="CtaBoyImage"
+        class="tw-h-full tw-w-[14.1875rem] tw-object-cover md:!tw-w-[22.64594rem] md:tw-w-[25.1875rem] lg:!tw-w-[32.625rem] xl:!tw-w-[34.625rem] xs:tw-w-[13.1875rem]"
+        alt="android-cta-image"
+      />
+    </div>
+    <div
+      class="tw-border-[#00000]/[0.6] tw-absolute tw-left-[10%] tw-z-0 tw-mr-[0.2rem] tw-mt-[2rem] tw-flex tw-flex-col tw-items-center tw-justify-center tw-rounded-lg tw-border-2 tw-py-[0.5rem] md:!tw-mt-[4rem] lg:!tw-mt-[6rem] xl:!tw-mt-[8rem] xl:tw-py-[2rem] 2xl:!tw-pb-[4rem] 2xl:!tw-pt-[3rem] 2xl:tw-pr-[2rem] xs:tw-mt-[3rem] xs:tw-py-4"
+    >
+      <div
+        class="tw-ml-auto tw-w-[38%] tw-font-inter-semibold tw-text-[1.125rem] tw-leading-[1.4625rem] tw-text-black-core/[0.87] md:tw-text-[2.4375rem] md:tw-leading-[3.15625rem] lg:!tw-w-[47%] lg:tw-text-[3.4375rem] lg:tw-leading-[5.15625rem] xs:tw-w-[50%]"
+      >
+        <span>Let’s talk about what we can build and scale together.</span>
```

Comment:
Maybe you need to make the font smaller in lg to xl.

Reply:
its according to image size.

Reply:
But it doesn't look good see, it covers the whole box

![cta1](https://github.com/canopas/canopas-website/assets/115449373/dc70d540-f133-4ede-b67c-63be7fc3988b)

Reply:
Also provide some line-height for lg device